summarize/Readme.md

```diff
@@ -2,17 +2,48 @@
 
 Summarize is a tool to produce a human readable summary of `measureme` profiling data.
 
-## Example
+## Installing summarize
+
+In order to use this tool you will first want to build it. To do this follow the steps
+below:
 
 ```bash
-$ git clone https://github.com/rust-lang/regex.git
+$ git clone https://github.com/rust-lang/measureme.git
+$ cd measureme/summarize
+$ cargo +nightly build --release
+```
+
+This will compile the `summarize` binary which we will need later. It will be located in
+the `measureme/target/release` folder (ie. not in the `summarize` folder).
```

Comment:
I found that `cargo +nightly install --release --path .` was convenient to use so you don't have to locate `summarize`.

Reply:
This was my idea since the profile format isn't stable. Given other discussions on Zulip, it may be fine to `cargo install` anyway. Let's see what @michaelwoerister thinks.

Reply:
I think ideally we would have a rustup component that contains all the self-profiling tools. That way they wouldn't go out of sync with the compiler. I'm not terribly familiar with rustup components though.

What exactly does `cargo +nightly install --release --path .` do?

Reply:
It pretty much just builds it and copies the executable to `~/.cargo/bin` so it's in your PATH.

Reply:
Oh, so the `--path .` allows to install crates that are not on `crates.io`? That would make sense.